### PR TITLE
LPS-185518 Create a SF check for replace with searchContainer.setResultsAndTotal

### PR DESCRIPTION
**Pull request by TamyrisTorres**
Ticket: https://issues.liferay.com/browse/LPS-185518

### CODE REVIEW

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSetResultsSetTotalMethodCheck.java [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+		while (setTotalMatcher.find()) {
+			String methodCall = JavaSourceUtil.getMethodCall(
+				content, setTotalMatcher.start());
+
+			String fullMethodCall = StringBundler.concat(
+				StringPool.NEW_LINE, methodCall, StringPool.SEMICOLON,
+				StringPool.NEW_LINE);
+
+			if (!fileName.endsWith(".jspf")) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSetResultsSetTotalMethodCheck.java:78]: @TamyrisTorres, shouldn't we include in this check the extension `.jsp` too? Or just check if the `fileName.endsWith(".java")`?

File: modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSetResultsSetTotalMethodCheck.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+		throws Exception {
+
+		String newContent = content;
+		Matcher setResultsMatcher = _setResultsPattern.matcher(content);
+
+		while (setResultsMatcher.find()) {
+			String methodCall = JavaSourceUtil.getMethodCall(
+				content, setResultsMatcher.start());
+
+			if (!fileName.endsWith(".jspf")) {
```
Comment (natocesarrego) [inline modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaSetResultsSetTotalMethodCheck.java:98]: @TamyrisTorres, shouldn't we include in this check the extension `.jsp` too? Or just check if the `fileName.endsWith(".java")`?

**Comment by TamyrisTorres** [comment]
ci:test:sf

**Comment by natocesarrego** [comment]
@TamyrisTorres, pull request approved, please forward it to Kevin. :wink:

**Pull request closed without merging**